community/contributing/pr_workflow.rst

```diff
@@ -30,12 +30,19 @@ The `repository on GitHub <https://github.com/godotengine/godot>`_ is a
 `Git <https://git-scm.com>`_ code repository together with an embedded
 issue tracker and PR system.
 
+.. note:: If you are contributing to the documention, its repository can
+          be found `here <https://github.com/godotengine/godot-docs>`_. 
+
 The Git version control system is the tool used to keep track of successive
 edits to the source code - to contribute efficiently to Godot, learning the
-basics of the Git command line is *highly* recommended. There exist some
-graphical interfaces for Git, but they usually encourage users to take bad
-habits regarding the Git and PR workflow, and we therefore recommend not to
-use them (especially GitHub's online editor).
+basics of the Git command line is *highly* recommended. 
+
+There exist some graphical interfaces for Git, but they usually encourage users 
+to take bad habits regarding the Git and PR workflow, and we therefore recommend 
+not to use them. Especially not the Github online editor. It leads to messy PRs filled 
+with too many commits. That being said, some text editors such as Visaul 
```

Comment:
I don't think this addition is useful. We say that we recommend using the command line over GUIs. Contributors are free to use them anyway if they're confident that their usage fits the expected workflow.

The only change I'd make on this paragraph is to change:

> (especially GitHub's online editor)

to:

> In particular, we advise not to use GitHub's online editor for code contributions (it's tolerated for documentation changes) as it enforces one commit per file and per modification, which quickly leads to PRs with an unreadable Git history (especially after peer review).